Dockerfile

```diff
@@ -8,6 +8,19 @@ ENV GOPATH=/go \
 RUN apk update && \
 	apk add git nodejs npm go ca-certificates make musl-dev bash icu-data
 
+# Get a recent specific version of golang
+ARG GOLANG_VERSION=1.19.3
+#we need the go version installed from apk to bootstrap the custom version built from source
+RUN apk update && apk add go gcc bash musl-dev openssl-dev ca-certificates && update-ca-certificates
+RUN wget https://dl.google.com/go/go$GOLANG_VERSION.src.tar.gz && tar -C /usr/local -xzf go$GOLANG_VERSION.src.tar.gz
+RUN cd /usr/local/go/src && ./make.bash
+ENV PATH=$PATH:/usr/local/go/bin
+RUN rm go$GOLANG_VERSION.src.tar.gz
+#we delete the apk installed version to avoid conflict
+RUN apk del go
+RUN go version
+# end installing golang.
```

Comment:
Should we instead use `golang:1.19` image for the build and do multi-stage builds like Nebraska instead?